debezium-core/src/main/java/io/debezium/converters/spi/CloudEventsProvider.java

```diff
@@ -31,6 +32,10 @@ public interface CloudEventsProvider {
      */
     RecordParser createParser(Schema schema, Struct record);
 
+    default RecordParser createParser(Schema schema, Struct record, Struct source, SchemaAndValue operation, Struct transaction) {
+        return createParser(schema, record);
+    }
```

Comment:
Can you explain why we explicitly only override this for PostgreSQL but don't do this for other connectors?

Reply:
This is because I could only test Debezium with Postgres in my project. But apparently a new test should be enough for other databases, so I'm going to add it